### DISCUSSION TITLE
Stuck at  2. 4: Endoscopic Models Deployment with MONAI Label

### DISCUSSION THREAD

**Discussion by BerkOlcay** [Project-MONAI/tutorials#1259] [Q&A]
Hello, I'm trying to go through the steps. I could follow until step 4. 2.4 Step 4: Endoscopic Models Deployment with MONAI Label. This step is to deploy MONAI Label plugin with endoscopic models using Nuclio tool. I created a workspace folder and then tried to run the command
 `# 1: If in Toolkit, users can get CVAT plugin folder from the toolkit 
docker docker cp $(docker container ls | grep 'nvcr.io/nvidia/clara/monai-toolkit' | awk '{print $1}'):/opt/conda/monailabel/plugins/cvat workspace` 
I recieve an error that is must specify at least one container source. I don't have any idea what the problem is. Could anyone please help me with that? Thanks.

**Comment by BerkOlcay** [comment]
can be deleted.

**Comment by tangy5** [comment]
Yeah, that doc is for some specific usecase when the Endoscopy app is used in the Toolkit docker. If you are using GIthub version, you could ignore that step. Thanks.

**Reply by BerkOlcay** [reply]
Thanks for the quick reply. I passed that part and was able to annotate using the model. Now, I'm stuck at 4.1 Start MONAI Label server for Active Learning. I modified the parts with my username and password and run the commands. I was expecting to see an active learning task. But MONAI Label server didn't automatically upload any images to CVAT. Server in terminal responds " No Tasks with completed status (current: None) to refresh/download the final labels". I should also mention that unzipped endoscopy_frames looks different than at part 1.3. So, I don't know what went wrong. Would you be able to help me out?

**Comment by tangy5** [comment]
The current sample data provided do not include validation set, we are going to prepare another sample data to split validation set by default. Could you follow the doc and manually set the valdiation set. 

For the automatic uploading process, if there are serveral setups exisiting, the program will consider those as already done..Can you try to do a clean restart. By firstly remove the datastore.json in the dataset and delete existing tasks in the CVAT.

Thank you.
